src/test/resources/fixtures/node-disallow-insecure-protocol/settings.gradle

```diff
@@ -0,0 +1,24 @@
+import org.gradle.api.initialization.resolve.RepositoriesMode
+import org.gradle.util.GradleVersion
+
+if (GradleVersion.current() >= GradleVersion.version("6.8")) {
+    dependencyResolutionManagement {
+        repositories {
+            repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+
+            ivy {
+                name = "Node.js"
+                setUrl("https://nodejs.org/dist/")
```

Comment:
This probably needs to be `http` but `nodejs.org` is not accessible over http, maybe just pointing it to http is enough, if we're lucky Gradle will complain in that it has to make a http requests in the first place and not be satisfied that it's a https-redirect.

Reply:
Let me try.

Reply:
Seems to work - Gradle follows the redirect to https if `allowInsecureProtocol` is `true` and barfs if it's `false`,

Reply:
that's great, makes the testing easy :D